modern/src/common/converter.js

```diff
@@ -1,13 +1,31 @@
-export const speedConverter = (value, unit) => {
+export const speedConverter = (value, unit, back = false) => {
```

Comment:
I think we should split functions. For example make `speedFromKnots` and `speedToKnots` and then keep the factor switch in some shared internal function.

Reply:
done